### DISCUSSION TITLE
different Multiple Artist separator in this repo and the library repo

### DISCUSSION THREAD

**Discussion by RoguedBear** [jojo2357/Music-Discord-Rich-Presence#31] [Q&A] [answered]
I was going through the README and saw the following syntax for `dat` files:
```
<executeable playing music name>=<player name>
id=<client ID from earlier>
Album Name 1==albumname1==Artist Name 1
Album Name 2==albumname2==Artist Name 2==Artist Name 3
Album Name 3==albumname3
```

Here multiple artists are separated by `==`. But in the [MDRP Library](https://github.com/jojo2357/Music-Discord-Rich-Presence-Library) multiple artists are separated by `/`

are both ways accepted?

**Comment by jojo2357** [comment, marked as answer]
Show me where multiple artists are separated by `/` so I can eradicate it. Never has been and never will be that way. Always `==`

**Reply by RoguedBear** [reply]
https://github.com/jojo2357/Music-Discord-Rich-Presence-Library/issues/10#issuecomment-808694417
![image](https://user-images.githubusercontent.com/39442192/113256219-810a4780-92e6-11eb-8ae1-5fbfafd599a9.png)

(The way `Lauv & LANY` is written is how it was in the metadata. that can be ignored)

**Reply by jojo2357** [reply]
I think thats just the artist because the library would never do that. Also the library doesnt actually handle multiple artists quite yet. Still ironing out improvements but im really busy rn sorry for the delay :/

**Reply by RoguedBear** [reply]
Its okay, the delay is not an issue. you can take your time bringing out improvements it is fine 😃 👍 

(Btw, if multiple artists will start using `==` in the library then I would also have to update the github actions bot to account for the `==`)
